app/assets/javascripts/discourse/app/controllers/json-schema-editor.js

```diff
@@ -0,0 +1,56 @@
+import Controller from "@ember/controller";
+import { create } from "virtual-dom";
+import { iconNode } from "discourse-common/lib/icon-library";
+import loadScript from "discourse/lib/load-script";
+import ModalFunctionality from "discourse/mixins/modal-functionality";
+import { schedule } from "@ember/runloop";
+
+export default Controller.extend(ModalFunctionality, {
+  editor: null,
+  jsonSchema: null,
+
+  onShow() {
+    loadScript("/javascripts/jsoneditor.js").then(() => {
+      schedule("afterRender", () => {
+        const el = document.querySelector("#json-editor-holder");
```

Comment:
This should be moved into a component, since we don't want controllers to access the DOM.

You could create a `{{json-editor}}` component perhaps?